webui/react/src/hooks/useModal/Model/useModalModelCreate.tsx

```diff
@@ -54,14 +64,27 @@ const DEFAULT_MODAL_STATE = {
   modelName: '',
   tags: [],
   visible: false,
+  workspace: '1',
 };
 
 const useModalModelCreate = ({ onClose }: Props = {}): ModalHooks => {
+  const { canCreateModelWorkspace } = usePermissions();
+  const isRbacEnabled = useFeature().isOn('model_rbac');
+  const loadableWorkspaces = useWorkspaces();
+  const location = useLocation().pathname;
+  const isWorkspace = location.includes('/workspaces');
+  const workspaceId = useMemo(
```

Comment:
nit: Is this a reliable way to determine whether we are in a workspace and set the ID. This seems prone to failure if we updated our `URL` scheme to either not include `/workspaces` or is the order of the workspace id in the `URL` changed. 

I think it seems more reliable for this information to be a prop passed into the component. However, I am open to this implementation if it is better.